addon/data/postmessageProxyFrame.js

```diff
@@ -0,0 +1,17 @@
+const TELEMETRY_QUEUE_POLL_INTERVAL = 5000;
+
+// Periodically poll for queued telemetry pings
+setInterval(function() {
+  document.getElementById('tp-proxy').contentWindow.postMessage({
+    op: 'fetchTelemetryPings'
+  }, '*');
+}, TELEMETRY_QUEUE_POLL_INTERVAL);
+
+// Listen for postMessage
+window.addEventListener('message', function(ev) {
```

Comment:
Same as above - the messages can come from one of prod / stage / dev / local and I don't have a way to configure that yet